### ISSUE TITLE
Cannot add a new language from settings

### ISSUE THREAD

**Issue by alshell7** [samtecspg/articulate#1034]
### Cannot add a new language from settings and use the same for the agent

- Pulled from the master - v0.31.1
- Windows 10 Pro - Version 1903
- Running Locally
- Firefox 75.0 (64-bit)

Noticing, that the languages can be added under the `Settings: System configuration` > `General` section. I assume that, the `Agent Languages` corresponds to Rasa NLU language support. And the UI language is only the locale configuration for the Articulate UI. Here are the steps I did:

* Updated the `Agent Languages` by adding Arabic:
```
[
  {
    "value": "de",
    "text": "Deutsch"
  },
  {
    "value": "en",
    "text": "English"
  },
  {
    "value": "es",
    "text": "Español"
  },
  {
    "value": "fr",
    "text": "Française"
  },
  {
    "value": "pt",
    "text": "Portugues"
  },
  {
    "value": "ar",
    "text": "Arabic"
  }
]
```
* Saved the settings and tried creating a new agent, selecting `Arabic` as the language from the ComboBox. But could not create the agent as the `Save` button did not work. And received the following message on the chat window instead:
> Error: There was an error creating your agent. 

* Checked the logs from the docker by supplying `docker logs articulate_api_1`, and found the following message:

```
ValidationError: child "language" fails because ["language" must be one of [de, en, es, fr, pt]]
```

I could not proceed hereafter. Is it that UI language needs to be updated to the respective Agent Language? Any help is appreciated. Thank you!

**Comment by wrathagom** [comment]
@alshell7 do Rasa and spaCy support arabic?

That setting really only applies to spaCy and duckling. and If you're using the default pipelines it only has an impact if you're using system entities from spaCy or Duckling.

@dcalvom did you type something up on adding a new language, I was thinking we had a guide for it someplace. Off the top of my head things that would need touched:

- UI menu
- API validation
- Rasa Docker image

**Comment by alshell7** [comment]
Yes [Rasa](https://github.com/RasaHQ/rasa/issues/1074) does support arabic. As from this [PR](https://github.com/RasaHQ/rasa/pull/1095). Usually using the Tensorflow pipeline, should get working as well. But it's again optional, the DIET Classifier for Intent classification and Duckling are a good catch together by default.

**Comment by wrathagom** [comment]
We are using version 0.14.4 of Rasa, that is the last version where they supported multiple models, which Articulate requires for now. the DIET classifier isn't an option here, as that's a fairly recent introduction.

**Comment by wrathagom** [comment]
we do have the tensorfzlow pipeline, which to use a different language with that nothing is required, just type your sayings in the language you want. The drop down only has an impact for space and duckling.

**Comment by rohetoric** [comment]
How to implement arabic for intent classification? I'm tired of running through PR issues and still not able to figure it out-
History of my navigation ->
PR #1095 says we can get Arabic support but here it @alshell7 says we cannot implement it? 
I have the rasa version - '1.10.2' .
The settings in the config.yml file show that we should involve tensorflow embeddings? How do we do that? Can I get an example of the config file to support arabic in this version of rasa ?  @wrathagom can you pls give this a look?
